reference/7.0/Microsoft.PowerShell.Core/About/about_Calculated_Properties.md

```diff
@@ -0,0 +1,506 @@
+  one or more properties. These are boolean values.
+
+The hashtable keys need not be spelled out as long as the specified name prefix
+is unambiguous. For example, `n` can be used in lieu of `Name` and `e` can be
+used in lieu of `Expression`.
+
+## Examples
+
+### Compare-Object
+
+With calculated properties, you can control how the properties of the input
+objects are compared.
+
+```powershell
+Compare-Object @{p=1} @{p=2} -property @{ Expression = { $_.p % 2 } }
+```
+
+```Output
+ $_.p % 2  SideIndicator
+---------- -------------
+         0 =>
+         1 <=
+```
+
+### ConvertTo-Html
+
+`ConvertTo-Html` can convert a collection of objects to an HTML table.
+Calculated properties allow you to control how the table is presented.
+
+```powershell
+Get-Alias |
+  ConvertTo-Html Name,
+                 Definition,
+                 @{n='ParamCount';e={$_.Parameters.Keys.Count};align='center'} |
+    Out-File .\aliases.htm -Force
+```
+
+This example creates an HTML table contain a list of PowerShell aliases and the
+number parameters for each aliased command. The values of **ParamCount** column
+are centered.
+
+### Format-Custom
+
+`Format-Custom` provides a custom view of an object in a format similar to a
+class definition. More complex objects can contain members that are deeply
+nested with complex types. The **Depth** parameter of `Format-Custom` specifies
+the depth of expansion for all properties. The `depth` attribute allows you to
+specify the depth of expansion per property.
+
+In this example we use the `depth` attribute to simplify the custom output for
+the `Get-Date` cmdlet . `Get-Date` returns a **DateTime** object. The **Date**
+property of this object is also a **DateTime** object, so the object is nested.
+
+```powershell
+Get-Date | Format-Custom @{e={$_.Date};depth=1},TimeOfDay
+```
+
+```Output
+class DateTime
+{
+  $_.Date =
+    class DateTime
+    {
+      Date = 8/7/2020 12:00:00 AM
+      Day = 7
+      DayOfWeek = Friday
+      DayOfYear = 220
+      Hour = 0
+      Kind = Local
+      Millisecond = 0
+      Minute = 0
+      Month = 8
+      Second = 0
+      Ticks = 637323552000000000
+      TimeOfDay = 00:00:00
+      Year = 2020
+      DateTime = Friday, August 07, 2020 12:00:00 AM
+    }
+  TimeOfDay =
+    class TimeSpan
+    {
+      Ticks = 435031592302
+      Days = 0
+      Hours = 12
+      Milliseconds = 159
+      Minutes = 5
+      Seconds = 3
+      TotalDays = 0.503508787386574
+      TotalHours = 12.0842108972778
+      TotalMilliseconds = 43503159.2302
+      TotalMinutes = 725.052653836667
+      TotalSeconds = 43503.1592302
+    }
+}
+```
+
+### Format-List
+
+In this example, we use calculated properties to change the name and format of
+the output from `Get-ChildItem`.
+
+```powershell
+Get-ChildItem ..\*.json -File |
+  Format-List Fullname,
+              @{
+                 name='Modified'
+                 expr={$_.LastWriteTime}
```

Comment:
Seems inconsistent to have `expression` written this way.

Reply:
Fixed